src/ussd_clinic_rapidpro.js

```diff
@@ -403,6 +401,9 @@ go.app = function() {
         });
 
         self.add("state_edd_month", function(name) {
+            if (typeof self.im.user.get_answer("state_active_subscription") != "undefined") {
```

Comment:
This check is a bit hard to read and there is a helper function for this that was used here: https://github.com/praekeltfoundation/ndoh-jsbox/pull/433/files#diff-5c4090be76401c5461c39da6838993894732990276768da57f543ead9240f2baL313